src/OctoshiftCLI.Tests/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandArgsTests.cs

```diff
@@ -70,6 +70,25 @@ public void It_Throws_When_Aws_Bucket_Name_Not_Provided_But_Aws_Access_Key_Provi
                 .WithMessage("*AWS S3*--aws-bucket-name*");
         }
 
+        [Fact]
+        public void It_Throws_When_Aws_Bucket_Name_Provided_With_UseGithubStorage_Option()
+        {
+            var args = new MigrateRepoCommandArgs
+            {
+                ArchivePath = ARCHIVE_PATH,
+                GithubOrg = GITHUB_ORG,
+                GithubRepo = GITHUB_REPO,
+                AzureStorageConnectionString = AZURE_STORAGE_CONNECTION_STRING,
```

Comment:
Azure storage connection string shouldn't be provided along with aws bucket name. We probably need and extra validation for when both aws and azure options are provided together but probably it'd be a scope creep for this PR. I guess we need to create a ticket to revise our validations and add missing ones.